.github/workflows/build-containers.yml

```diff
@@ -61,134 +52,235 @@ jobs:
+              ref: context.ref,
+              inputs: {
+                image: "${{ matrix.build_config.image }}",
+                base_name: "${{ matrix.build_config.base_name }}",
+                base_version: "${{ matrix.build_config.base_version }}",
+                base_repo: "${{ matrix.build_config.base_repo }}"
+              }
+            });
+            console.log(result);
+
+  trigger-default-builds:
+    needs: check-config
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        build_config: ${{ fromJson(needs.check-config.outputs.default_image_matrix) }}
+    steps:
+
+      - name: Trigger standard build workflow (default)
+        if: ${{ matrix.build_config.build_type == 'standard' }}
+        uses: actions/github-script@v4
         with:
-          repo: testing
-          osg_series: ${{ matrix.series.name }}
-          context: ${{ matrix.image }}
-          base_os: ${{ matrix.series.os }}
-          registry_url: docker.io
-          registry_user: ${{ secrets.DOCKER_USERNAME }}
-          registry_pass: ${{ secrets.DOCKER_PASSWORD }}
-
-      - name: Push to Docker Hub (${{ matrix.series.name }}-release)
-        if: always()
-        uses: opensciencegrid/push-container-action@main
+          script: |
+            const result = await github.actions.createWorkflowDispatch({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              workflow_id: 'standard-build.yml',
+              ref: context.ref,
+              inputs: {
+                image: "${{ matrix.build_config.image }}",
+                base_name: "${{ matrix.build_config.base_name }}",
+                base_version: "${{ matrix.build_config.base_version }}",
+                base_repo: "${{ matrix.build_config.base_repo }}"
+              }
+            });
+            console.log(result);
+
+      - name: Trigger repo build workflow (default)
+        if: ${{ matrix.build_config.build_type == 'repo' }}
+        uses: actions/github-script@v4
         with:
-          repo: release
-          osg_series: ${{ matrix.series.name }}
-          context: ${{ matrix.image }}
-          base_os: ${{ matrix.series.os }}
-          registry_url: docker.io
-          registry_user: ${{ secrets.DOCKER_USERNAME }}
-          registry_pass: ${{ secrets.DOCKER_PASSWORD }}
+          script: |
+            const result = await github.actions.createWorkflowDispatch({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              workflow_id: 'repo-build.yml',
+              ref: context.ref,
+              inputs: {
+                image: "${{ matrix.build_config.image }}",
+                base_name: "${{ matrix.build_config.base_name }}",
+                base_version: "${{ matrix.build_config.base_version }}",
+                base_repo: "${{ matrix.build_config.base_repo }}"
+              }
+            });
+            console.log(result);
```

Comment:
This job feels unnecessary if you flatten the concept of the per-image vs default config files: you just have in-repo vs external-repo builds and the fact that a bunch of the former came from the default config file is irrelevant at this point of the workflow